pkg/validate/providers_test.go

```diff
@@ -0,0 +1,49 @@
+package validate
+
+import (
+	"github.com/armory/spinnaker-operator/pkg/accounts/kubernetes"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha2"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestGetAccountsFromConfig(t *testing.T) {
+	spinsvc := &v1alpha2.SpinnakerService{
```

Comment:
There are already helper functions in `test` module to load test manifest yaml files, so you don't need to code the whole `SpinnakerService` structure in each test case:
```
spinsvc := test.ManifestToSpinService(manifestFilePath, t)
```
Also, it makes it easier to replicate bugs when you have a sample manifest file to include quickly in a new test case.
We can follow the same approach for tests related with account CRD's.

Reply:
I see the value for larger manifests but I find that much less readable. It's also hard to see where the file is used. We should probably add as a simple parse with a given string. What do you think?

Reply:
Parse with a given string is ok for smaller structures.